apps/selvbetjening-ui/src/common/NavigasjonMeny/NavigasjonMeny.tsx

```diff
@@ -23,9 +24,10 @@ export const NavigasjonMeny = ({ tilbakePath, skalSendeSoeknad, onNeste }: Props
     return (
         !!innhold && (
             <VStack gap="4">
-                <BodyShort as="div" size="small" textColor="subtle">
-                    {innhold.navigasjonMeny?.sistLagret?.[spraak]}: 10. mars 2024 kl. 13.55
-                </BodyShort>
+                {/* TODO: kommenterer ut disse til vi funksjonaliteten for de på plass*/}
+                {/*<BodyShort as="div" size="small" textColor="subtle">*/}
+                {/*    {innhold.navigasjonMeny?.sistLagret?.[spraak]}: 10. mars 2024 kl. 13.55*/}
+                {/*</BodyShort>*/}
```

Comment:
Er det ikke bedre å bare fjerne? Slik den er nå må den uansett endres en del på for å få inn dataene riktig, og man kan bare se på historikken for å finne koden hvis man trenger den